web/web.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2029 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package web
+
+import (
+	"image/color"
+	"net/http"
+)
+
+// Config represents the configuration of the web listener.
+type Config struct {
+	Color       color.Color // Used for the landing page header
```

Comment:
Does this need to be configurable?

Reply:
Thanks for the very early review. I think that having just a different color for each exporter would be a nice 'touch'; without going to the point where we offer themes.

Reply:
Yes, it's nice idea. For example, using the MySQL brand colors for the mysqld_exporter. :smile: